movement/movement.c

```diff
@@ -644,4 +643,8 @@ void cb_tick(void) {
     } else {
         movement_state.subsecond++;
     }
+    if (event.event_type != EVENT_NONE) {
+        return;
+    }
+    event.event_type = EVENT_TICK;
```

Comment:
The PR should contain these lines. The watch face that reproduces this issue is of great interest to developers but unlikely to be interesting to users. Perhaps it should be a separate PR or branch.

Is there a way to update `event` atomically?